x-pack/solutions/observability/plugins/synthetics/server/routes/filters/filters.ts

```diff
@@ -87,31 +91,31 @@ export const getSyntheticsFilters: SyntheticsRestApiRouteFactory<MonitorFiltersR
 const aggs = {
   monitorTypes: {
     terms: {
-      field: `${monitorAttributes}.${ConfigKey.MONITOR_TYPE}.keyword`,
+      field: `${syntheticsMonitorAttributes}.${ConfigKey.MONITOR_TYPE}.keyword`,
```

Comment:
This is me not knowing enough about saved objects probably.. I see that we're fetching 2 different types of saved object, the legacy one and the new one.. do the attributes depend on the type of the retrieved saved object?